arcgis-ios-sdk-samples/Augmented reality/Display scenes in tabletop AR/README.md

```diff
@@ -2,7 +2,7 @@
 
 Use augmented reality (AR) to pin a scene to a table or desk for easy exploration.
 
-![Scene content shown sitting on a surface, as if it were a 3D printed model](image1.png)
+![Scene content shown sitting on a surface, as if it were a 3D printed model](display-scene-tabletop.jpeg)
```

Comment:
I don't see any sample uses jpeg as title image. Do we have a convention for title image formats?

Also the new image is a bit blurry.

Reply:
Other SDKs use jpeg but I'm not sure if we have an iOS specific convention. I had to take this picture with an iPad and then resize is why it's blurry and why it's a jpeg. Any suggestions on how to resize the image while maintaining the quality?

Reply:
AidDrop the image to your Mac and resize it with Preview. Please use png for images.

Reply:
@philium I've updated the image to be png and airdropped and resized as you said but the image is still blurry.

Reply:
It seems you captured that screenshot with an iPad? Maybe increase the resolution a to ~480x640? I don't have a nice round table though, otherwise I can help to capture the screenshot. 😓